api-gateway-response/constants.go

```diff
@@ -24,12 +24,12 @@ type ErrorResponse struct {
 	Code    string `json:"code"`
 }
 
-type SuccessResponseBody struct {
+type SuccessResponse struct {
 	Results  any `json:"results"`
 	Metadata any `json:"metadata"`
 }
 
 type PaginationMetadata struct {
-	PageCount   int `json:"max_page"`
-	ResultCount int `json:"results_per_page"`
+	PageCount   int `json:"page_count"`
+	ResultCount int `json:"result_count"`
```

Comment:
hmm, since go-common na ito, maganda siguro kung magkaron ng concensus sa metadata format, or kung multiple formats when to use which, may parang 2-3 get all endpoints ata ako na ganito yung format ng metadata 😅 
```
	"metadata": {
		"total_items": 1,
		"max_page": 1,
		"current_page": 1
	}
```